src/otx/algorithms/segmentation/adapters/mmseg/configurer.py

```diff
@@ -534,6 +539,25 @@ def _configure_dataloader(cfg: Config) -> None:
 
         _configure_dataloader(cfg)
 
+    @staticmethod
+    def configure_input_size(
+        cfg, input_size_config: InputSizePreset = InputSizePreset.DEFAULT, model_ckpt: Optional[str] = None
+    ):
+        """Change input size if necessary."""
+        input_size = get_configurable_input_size(input_size_config, model_ckpt)
+        if input_size is None:
+            return
+
+        # segmentation models have different input size in train and val data pipeline
+        base_input_size = {
+            "train": 512,
+            "val": 544,
+            "test": 544,
+        }
```

Comment:
Wow... same for SegNext, too?

Reply:
yes, all segmentation refer same data pipeline file.